mmhuman3d/core/visualization/visualize_keypoints2d.py

```diff
@@ -440,7 +441,7 @@ def visualize_kp2d(
                 Defaults to None.
         img_format (str, optional): input image format. Default to '%06d.png',
         start (int, optional): start frame index. Defaults to 0.
-        end (int, optional): end frame index. Defaults to -1.
+        end (int, optional): end frame index. Defaults to None.
```

Comment:
Need to indicate that the end frame index is `exclusive`.

Reply:
already solved. Our `end` serves the same as `list`, `tensor`, and `numpy` slice.